### DISCUSSION TITLE
Meeting Titles Should Be Sorted in Alpha Order

### DISCUSSION THREAD

**Discussion by Grateful-Viv** [code4recovery/12-step-meeting-list#648] [Q&A]
I have two meetings on Friday night at 6PM for the same group. One is in-person and the other is online (it was just published). They both start with the same title (Shore Serenity). In the listing they should show up one after the other. Yet they do not. In between is another meeting also at 6PM on Friday. I am sure it's because of the order it was published. It should be the order of day, time, and then title (name).

It looks like this
Shore Serenity Discussion
Friday Night Ladies Happy Hour
Shore Serenity Online

It should like this
Friday Night Ladies Happy Hour
Shore Serenity Discussion
Shore Serenity Online

![Shore Serenity](https://user-images.githubusercontent.com/62247421/150044435-a155c50f-b06f-41d0-aed0-6fce7e78ba2b.png)

Update: Seems it's not the publish date or the update date that determines the sort order, because I changed it and it had no effect. How can I get these meetings for the same day of week at the same time show up in alpha order by permalink name or meeting name?

**Comment by Grateful-Viv** [comment]
I would appreciate a response to this. I even downloaded the CSV, changed the order of the rows so the two meetings from the one group would be one after the other, and then uploaded the meetings back. It still shows up with the other meeting in between.

Thank you.

**Comment by joshreisner** [comment]
hi @Grateful-Viv the sort order currently is:

1. day
2. time
3. location
4. name

to group meetings at the same location together. this logic was added years ago, before online meetings support.

import order does not have an impact

**Reply by Grateful-Viv** [reply]
Thanks for the response @joshreisner. Would it be possible to change this order to 

1. day
2. time
3. name
4. location

What I ended up doing so it could sort properly was added TC to the online meeting. But of course, now it says location temporarily closed, when there is an actual live in-person meeting going on at that location in addition to the online meeting.

**Reply by joshreisner** [reply]
@97eighty @brianw-area24 @tech2serve any thoughts on this proposal?

**Reply by tech2serve** [reply]
I agree that the situation @Grateful-Viv is seeing is less than ideal.  I was trying to think through implications of a wide-scale change, in a scenario where two different locations have meetings of the same name, which would now be grouped together. 

For example, Church A that has a newcomer’s meeting and an open discussion at the same time, and one of the meeting names is “Shore Serenity” (hypothetical example). This might bust up the logical display of the two meetings at Church A.

Is that a big deal/risk? I do see a lot of meetings with the same name around an area, but most already differentiate with day/time, so probably not a huge risk.

If @code4recovery/tsml-maintainers agree, let’s convert this to an issue and discuss options to implement it.
